deploy/deploy_research_environment.yml

```diff
@@ -25,6 +25,71 @@
         regexp: ClientAliveCountMax
         line: ClientAliveCountMax=384
 
+    - name: additional symlinks for odbc.ini, odbcinst.ini, freetds, cups to shared env
+      file:
+        state: link
+        src: "{{ item.src }}"
+        dest: "{{ item.dest }}"
+      with_items: "{{ additional_symlinks }}"
```

Comment:
@olivermel This should be a bit more descriptive, like "research_environment_links"